src/components/views/dialogs/ShareDialog.tsx

```diff
@@ -155,7 +155,7 @@ export default class ShareDialog extends React.PureComponent<IProps, IState> {
             checkbox = <div>
                 <StyledCheckbox
                     checked={this.state.linkSpecificEvent}
-                    onClick={this.onLinkSpecificEventCheckboxClick}
+                    onChange={this.onLinkSpecificEventCheckboxClick}
```

Comment:
This change is unrelated to what you are fixing here, please don't conflate your PRs

Reply:
i don't know why the commit is still showing even though i opened a new branch with this new PR

Reply:
You probably branched straight from your other branch, instead of `develop` - easy way around that in future is to always switch back to develop first.

Reply:
> You probably branched straight from your other branch, instead of `develop` - easy way around that in future is to always switch back to develop first.

Noted, Thanks, i am learning a lot from you : )